apps/frontend/public/service-worker.js

```
@@ -27,37 +82,40 @@
+    const isDynamicRoute = DYNAMIC_ROUTES.includes(event.request.url)
 
-            return fetch(event.request)
-                .then((networkResponse) => {
-                    if (networkResponse && networkResponse.status === 200) {
-                        const clonedResponse = networkResponse.clone()
+    if (isStaticRoute) {
+        event.respondWith(
+            caches.match(event.request).then((cachedResponse) => {
+                if (cachedResponse) {
+                    const currentTime = Date.now()
+                    const cachedTime = parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)